AllocatorTest.hpp

```diff
@@ -81,7 +81,7 @@ struct STLAllocator {
   typedef T value_type;
   typedef std::size_t size_type;
 
-  typedef DynamicPoolAllocator<AllocatorType> PoolType;
+  typedef DynamicSizePool<AllocatorType> PoolType;
```

Comment:
We will need to make sure and change all of our using code to use this new name.